controller/otpController.js

```diff
@@ -6,11 +6,63 @@ const {
 } = require("../utils/customResponse.js");
 const constants = require("../constants/constants.js");
 
+// exports.test = async (req, res) => {
+//   const { number } = req.body;
+//   try {
+//     client.verify.v2
+//       .services("VA9391ad1effed1b94b84b0ba299a6faf4")
+//       .verifications.create({ to: number, channel: "sms" })
+//       .then((verification) =>
+//         res.status(200).json(
+//           successResponse({
+//             status: verification.status,
+//             message: "Otp sent successfully",
+//           })
+//         )
+//       );
+//   } catch (error) {
+//     res
+//       .status(404)
+//       .json(
+//         customResponse(constants.SUCCESS_CODE, "Failed to send OTP", error)
+//       );
+//   }
+// };
+
+// exports.testVerify = async (req, res) => {
+//   const { code } = req.body;
+//   try {
+//     const verificationCheck = await client.verify.v2
+//       .services("VA9391ad1effed1b94b84b0ba299a6faf4")
+//       .verificationChecks.create({ to: "+917416279050", code });
+
+//     return res.status(200).json(
+//       successResponse({
+//         status: verificationCheck.status,
+//         message: "Verified Successfully",
+//       })
+//     );
+//   } catch (error) {
+//     res
+//       .status(404)
+//       .json(
+//         customResponse(constants.FAILURE_CODE, "OTP verification Failed", error)
+//       );
+//   }
+// };
+
```

Comment:
Can you please remove this commented code
I don't think we need this anymore right since it has already been tested with the new alternative